test/suites/integration/httproute_header_match_test.go

```diff
@@ -18,6 +18,11 @@ import (
 )
 
 var _ = Describe("HTTPRoute header matches", func() {
+
+	AfterEach(func() {
+		testFramework.CleanTestEnvironment(ctx)
+	})
+
```

Comment:
do you need to add this AfterEach()?

Reply:
Yes I was having resources leak on this previously.